contracts/interchain-token-service/tests/execute.rs

```diff
@@ -111,8 +111,10 @@ fn register_multiple_chains_succeeds() {
         .map(|i| msg::ChainConfig {
             chain: i.to_string().parse().unwrap(),
             its_edge_contract: i.to_string().parse().unwrap(),
-            max_target_decimals: 18u8,
-            max_uint: Uint256::MAX.try_into().unwrap(),
+            truncation: TruncationConfig {
+                max_decimals_when_truncating: 18u8,
+                max_uint: Uint256::MAX.try_into().unwrap(),
```

Comment:
add test coverage (chains with differing values)? no test is failing on a logic change lol